src/main/java/com/conveyal/datatools/manager/jobs/MergeFeedsJob.java

```diff
@@ -0,0 +1,571 @@
+     * @param table table to merge
+     * @param feedsToMerge map of feedSources to zipFiles from which to extract the .txt tables
+     * @return single merged table for feeds or null if the table did not exist for any feed
+     */
+    private byte[] constructMergedTable(Table table, List<FeedToMerge> feedsToMerge) throws IOException {
+        String keyField = table.getKeyFieldName();
+        String orderField = table.getOrderFieldName();
+        if (mergeType.equals(MTC)) {
+            // MTC requires that the stop and route records be merged based on different key fields.
+            switch (table.name) {
+                case "stops":
+                    keyField = "stop_code";
+                    break;
+                case "routes":
+                    keyField = "route_short_name";
+                    break;
+                default:
+                    // Otherwise, use the standard key field (see keyField declaration.
+                    break;
+            }
+        }
+        // Set up objects for outputting the data and tracking the rows encountered
+        ByteArrayOutputStream tableOut = new ByteArrayOutputStream();
+        Map<String, String[]> rowValuesForKeys = new HashMap<>();
+        Set<String> rowStrings = new HashSet<>();
+        int mergedLineNumber = 0;
+        // Get the spec fields to export
+        List<Field> specFields = table.specFields();
+        boolean stopCodeMissingFromFirstTable = false;
+        try {
+            // Iterate over each zip file.
+            for (int f = 0; f < feedsToMerge.size(); f++) {
+                mergeFeedsResult.feedCount++;
+                // FIXME add check for merge type.
+                if (f > 0 && (table.name.equals("agency") || table.name.equals("feed_info")) && mergeType.equals(MTC)) {
+                    // Always prefer future file for agency and feed_info tables, which means that we can skip
+                    // iterations following the first one.
+                    // FIXME: This could cause issues with routes or fares that reference an agency_id that no longer
+                    //  exists.
+                    LOG.warn("Skipping {} file for feed {}/{} (future file preferred)", table.name, f, feedsToMerge.size());
+                    continue;
+                }
+                FeedToMerge feed = feedsToMerge.get(f);
+                FeedVersion version = feed.version;
+                FeedSource feedSource = version.parentFeedSource();
+                // Generate ID prefix to scope GTFS identifiers to avoid conflicts.
+                String idScope = getCleanName(feedSource.name) + version.version;
+                CsvReader csvReader = JdbcGtfsLoader.getCsvReader(feed.zipFile, table, null);
+                // If csv reader is null, the table was not found in the zip file. There is no need to handle merging
+                // this table for the current zip file.
+                if (csvReader == null) continue;
```

Comment:
Please add a log.warn statement indicating that the table was not found in the zip file.